RLTest/__main__.py

```diff
@@ -287,6 +288,28 @@ def takeEnvDown(self, fullShutDown=False):
                     self.testsFailed.add(self.currEnv)
```

Comment:
I do not get it, sometimes you put a env in testsFailed and sometimes a touple?

Reply:
I forgot to add that line. It's always supposed to be `[test, [list of failures]]`